### PR TITLE
BACKEND-325: Dockerization and Post Feed Endpoint w/ Mockdata

### PR DESCRIPTION
**Pull request by kaanyolcu22**
I have configured PostgresSql, wrote the DockerFile and docker_compose.yml from the start with @arastasci . Then, I prepared a mockdata for the post feed page. Can you check if the mockdata is okay ? @Elifnurdeniz @yunusemreozdemir

### CODE REVIEW

**Comment by kaanyolcu22** [comment]
If you prefer more comments or a different endpoint for making comment or creating post let me know

**Comment by arastasci** [comment]
the endpoint with mock data looks good to me, good work @kaanyolcu22 - any review/feedback for the docker compose configuration we created would be appreciated.

**Comment by Elifnurdeniz** [comment]
I guess bookmark count is unnecessary. Also, it is better if we make the category (tags) field a list since a content can have multiple tags. Can you name it tags, also?

File: backend/app/views.py [hunk 1]
```diff
@@ -1,30 +1,183 @@
...
+      },
+      "post": {
+        "title": "Ephemeral: Short-lived",
+        "content": "Ephemeral means lasting for a very short time. Example: 'The beauty of the sunset was ephemeral but unforgettable.'",
+        "category": "Vocabulary",
+        "timestamp": "30 minutes ago"
+      },
+      "engagement": {
+        "likes": 50,
+        "comments": 8,
```
Comment (YavizGuldalf) [inline backend/app/views.py:43]: It says that there are 8 comments here, but below I can only see 3.

File: backend/app/views.py [hunk 2]
```diff
@@ -1,30 +1,183 @@
...
+      },
+      "post": {
+        "title": "Petrichor: The Smell of Rain",
+        "content": "Petrichor is the pleasant, earthy smell that comes after rain. Example: 'She loved the petrichor that filled the air after the storm.'",
+        "category": "Vocabulary",
+        "timestamp": "5 hours ago"
+      },
+      "engagement": {
+        "likes": 90,
+        "comments": 20,
```
Comment (YavizGuldalf) [inline backend/app/views.py:93]: It says 20 comments here but I can only see 3 below.

File: backend/app/views.py [hunk 3]
```diff
@@ -1,30 +1,183 @@
...
+      },
+      "post": {
+        "title": "Ineffable: Beyond Words",
+        "content": "Ineffable means too great or extreme to be expressed in words. Example: 'The beauty of the mountains was ineffable.'",
+        "category": "Vocabulary",
+        "timestamp": "4 hours ago"
+      },
+      "engagement": {
+        "likes": 80,
+        "comments": 18,
```
Comment (YavizGuldalf) [inline backend/app/views.py:143]: Once again, there are only 3 comments instead of 18.

**Review by YavizGuldalf — CHANGES_REQUESTED** [review]

**Comment by kaanyolcu22** [comment]
Sory , I mistakenly forgot it. Now I fixed it.

**Comment by arastasci** [comment]
@YavizGuldalf @Elifnurdeniz can you guys check if the revision is as you've requested?

**Review by YavizGuldalf — APPROVED** [review]
LGTM, but we must remember to change the 'category' field to 'tags'

**Comment by arastasci** [comment]
> LGTM, but we must remember to change the 'category' field to 'tags'

sure thing, opening the issue rn.

**Pull request merged**